tests/flakyfunctional/cylc-show/00-simple/flow.cylc

```diff
@@ -11,8 +11,8 @@
     [[graph]]
         PT1H = """
             bar => foo
-            foo:start => SHOW
-            SHOW:finish-all => end
+            foo:start => SHOW?
+            SHOW:finish-all? => end
```

Comment:
Not gonna lie I find this a bit confusing. But we can resolve the question of whether `finish` needs a `?` in a follow-up

Reply:
I'm somewhat expecting to be voted down on this, but let's argue it out to get the best result!

Reply:
https://github.com/cylc/cylc-flow/issues/4402